storybook/package.json

```diff
@@ -34,6 +34,6 @@
     "start": "storybook dev -p 6006",
     "build-and-serve": "yarn build & yarn serve",
     "build": "NODE_ENV=production storybook build",
-    "serve": "serve"
+    "serve": "serve ./storybook-static"
```

Comment:
Was there an issue with this command? The `serve` tool config is in the `serve.json` file and refers to this folder already.

Reply:
exactly, `yarn serve` would crash bc it wasn't finding an `index.html` unless I specified the filepath.

Reply:
fwiw, here is the output I get
![Screenshot 2024-09-12 at 8 59 42 AM](https://github.com/user-attachments/assets/bb2a86bf-4f8a-4b3a-8d48-704bc787eef0)

Reply:
The only way I can reproduce this is by running the `build-and-serve` command and I quickly noticed why it happened: I forgot the second **&** character and the tasks are started in parallel instead of consecutive order 🤦 

Can you fix this command and run it again, or run the build + serve commands manually one after another.
I just ran both commands from the project-root and from within the storybook folder and had no issues.

Reply:
Another reason I could imagine is that your system is picking up a globally installed version of `serve` which does not support the `serve.json` file yet. Is `serve -v` showing the version specified in the `package.json`?

Reply:
@misama-ct I spent _entirely_ too long trying to understand what was causing different behavior on my machine, and I've found that what's causing the issue is that `serve` has a dependency `serve-handler`, which has a dependency on `minimatch:3.1.2`.

In our root package.json, we have a [resolution](https://github.com/commercetools/ui-kit/blob/fba85949e7e95ff84a9f4a874ff8c4a3931c4aab/package.json#L173) to set `minimatch` to always resolve to `^9.0.0`.

The syntax used to call `minimatch` in `serve-handler` is valid in `3.1.2`, but not in `9.x`, thus the error I am seeing.

We resolve the `minimatch` version to greater than `9.x` because older versions have security vulnerabilities, and from what I can tell there is no way to tell `yarn v3` to resolve `3.1.2` only for `serve-handler`.

I verified that this issue also occurs for @tylermorrisford when he installs `main` and attempts to run `yarn serve`.


So out of curiosity, how are you building packages for the monorepo? because I cannot get this to work without modifying something.

Reply:
@misama-ct I'm going to revert the change for the purposes of this PR - it can be handled separately if necessary.